src/EnergyPlus/StandardRatings.cc

```diff
@@ -3097,7 +3133,7 @@ namespace StandardRatings {
             EIRFlowModFac(spnum) = Curve::CurveValue(state, EIRFFlowCurveIndex(spnum), AirMassFlowRatioRated);
 
             {
-                if (state.dataCurveManager->PerfCurve(EIRFTempCurveIndex(spnum)).numDims == 1) {
+                if (state.dataCurveManager->PerfCurve(EIRFTempCurveIndex(spnum))->numDims == 1) {
```

Comment:
Better to do `auto *eirFTempCurve` and call methods rather than do the curve lookup four separate times?